.env

```diff
@@ -0,0 +1,2 @@
+TORII_URL=127.0.0.1:1337
```

Comment:
I would like to have config as a JSON file called config.json. There will be another JSON file called trusted_peers.json as well (which will have the public keys and IPs of trust peers to bootstrap syncing with the network).

Reply:
As you wish)